scripts/util.py

```diff
@@ -820,6 +822,64 @@ def to_hostlist(nodenames) -> str:
     return hostlist
 
 
+def to_hostlist_fast(names: Iterable[str]) -> str:
```

Comment:
I am little skeptic about using to_hotlist_fast naming convention. At this point since we have 2 functions it makes sense but I think in the future we would update this back to to_hotlist when we remove legacy function.

Reply:
Yes, there is a TODO:

```
# TODO: replace with to_hostlist_fast
def to_hostlist(nodenames) -> str:
```